### PR TITLE
feature(main): add pyament deploy

### PR DESCRIPTION
**Pull request by cuisongliu**
Signed-off-by: cuisongliu <cuisongliu@qq.com>

### CODE REVIEW

File: controllers/user/deploy/README.md [hunk 1]
```diff
@@ -8,10 +8,16 @@ sealos build -t docker.io/labring/sealos-user-controller:dev -f Dockerfile .
...
+|`WechatPrivateKey`|Wechat private key|``|
+|`MchID`|Wechat mch id|``|
+|`MchCertificateSerialNumber`|Wechat mch certificate serial number|``|
+|`MchAPIv3Key`|Wechat mch api v3 key|``|
+|`AppID`|Wechat app id|``|
 
 ### How to run
 
 ```shell
+sealos run docker.io/labring/kustomize:v4.5.6
```
Comment (fanux) [inline controllers/user/deploy/README.md:21]: seems we don't need the callback URL anymore.

Comment (cuisongliu) [reply controllers/user/deploy/README.md:21]: I changed it to `NotifyCallbackURL`. It is actually a callback interface for payment to inform us of the status. I think it's more appropriate to keep it.

Comment (fanux) [reply controllers/user/deploy/README.md:21]: We don't use Notify too, controller will check the payment status every 0.3s, so needn't any notify

Comment (fanux) [reply controllers/user/deploy/README.md:21]: > We don't use Notify too, controller will check the payment status every 0.3s, so needn't notify

Comment (cuisongliu) [reply controllers/user/deploy/README.md:21]: > We don't use Notify too, controller will check the payment status every 0.3s, so needn't any notify

OK，Let me fix this problem.

**Review by fanux — APPROVED** [review]

**Pull request merged**